index.html

```diff
@@ -59,6 +59,171 @@ <h2>I'm a software <br> developer</h2>
                     <img src="img/Disabled.png">
                 </a>
             </div>
-        </section>   
+        </section>
+        <!-- The work section -->
+        <section class="works-section">
+            <div class="text">
+                <h2>My Recent <br> Works</h2>
+                <hr>
```

Comment:
- [ ] No comment @Ouail-01 , you have done an amazing job, I really appreciate it, your code is well structured, however, you have some few changes to do in the design before you project get approved, let's take a look them  🤗

<img width="700" alt="oua1" src="https://user-images.githubusercontent.com/77843050/205413013-ffb3b563-6139-443b-95c5-cdce9d971911.png">

- [ ] As highlighted, let me be more explicit.... at the first look on the background color of your app compared to the figma at the right, am sure to remark a difference between them, please make sure they match same as per the requirement
- [ ] the `multi-post` title, please use the same color and font as per the requirement.
- [ ] same as the technologies sections, kindly modify them to match the same design as in the figma